src/rez/plugin_managers.py

```diff
@@ -259,8 +265,43 @@ def register_plugin():
             'rezplugins' is always found first.
     """
     def __init__(self):
+        self._plugin_types_root = None
         self._plugin_types = {}
 
+    def locate_main_rezplugins(self):
```

Comment:
Let's not do this, and keep it simple instead. We expect a topmost 'rezplugins' submodule within the plugin package, and that's all.

Reply:
Oh sorry, this and the "MAIN" file were for fixing #960. Although #960 only happens on old-style plugins, but I think it's still a bit related to this PR since we are gonna implementing a new plugin installation method.

Reply:
reverted.